sycl/doc/extensions/C-CXX-StandardLibrary/DeviceLibExtensions.rst

```diff
@@ -32,3 +32,159 @@ Arguments:
 Example of a message:
 .. code:
    foo.cpp:42: void foo(int): global id: [0,0,0], local id: [0,0,0] Assertion `buf[wiID] == 0 && "Invalid value"` failed.
+
+cl_intel_devicelib_math
+==========================
```

Comment:
Do you plan to support them as part of this extension later? If so,
then you probably should mention it here.